region/lowernorfair/east.json

```diff
@@ -6086,13 +6086,22 @@
                   "note": "Requires one jump off a crumble block"
                 },
                 {
-                  "name": "MickeyMouse Crumbleless MidAir Spring Ball",
+                  "name": "Mickey Mouse Spring Ball Escape",
```

Comment:
This name is too similar to the crumblejump + springball escape.  HiJump+Springball is similar too.